docs/releases/5.2.1.txt

```diff
@@ -15,3 +15,7 @@ Bugfixes
 
 * Fixed a regression in Django 5.2 that caused unnecessary queries when
   prefetching nullable foreign key relationships (:ticket:`36290`).
+
+* Fixed a data loss possibility where ``file_move_safe()``, with
+  ``allow_overwrite`` set to ``True``, did not truncate the overwritten file
+  content (:ticket:`36298`).
```

Comment:
I would suggest a slight rewording to this since there is no true data loss, "only" data corruption (the data is always there and to some extent can be retrieved):
```suggestion
* Fixed data corruption in ``file_move_safe()`` when ``allow_overwrite=True``,
   where leftover content from a previously larger file could remain after
   overwriting with a smaller one due to lack of truncation (:ticket:`36298`).
```